src/main/webapp/testScript.js

```diff
@@ -0,0 +1,22 @@
+//button styling 
+mdc.ripple.MDCRipple.attachTo(document.querySelector('.foo-button'));
+
+//this funciton will implement the fetch to grab the list of comments from the database and call to the newDiv()
+//function to place the comments into the feed div.
+window.onload = function display_feed(){
+    
+}
+
+//creates a new div for each new comment
+function newDiv(){
+    //make this list by calling a fetch function 
```

Comment:
we want to be adding both the message of the comment as well as the nickname of the user

Reply:
Space for this will be added